static/app/views/issueList/filters.tsx

```diff
@@ -42,12 +42,21 @@ const SearchContainer = styled('div')`
 `;
 
 const StyledPageFilterBar = styled(PageFilterBar)`
-  flex: 0 1 0;
+  display: flex;
+  flex-basis: content;
   width: 100%;
-  max-width: 45rem;
-
-  > div > button {
-    width: 100%;
```

Comment:
@0Calories Looks like this deletion caused the flex issue. Restoring this block fixes the issue.
<img width="703" alt="image" src="https://user-images.githubusercontent.com/44172267/229592387-1ed2cb57-94d0-4e36-b61c-159387766101.png">

Reply:
Ah gotcha, I thought your concern was with the dropdowns being stretched out rather than their normal width. Yeah, restoring that CSS rule fixes the width of the issue category dropdown, thanks 👍